build-packages/scandipwa-template-plugin-system/util/exposed-api-keys.js

```diff
@@ -0,0 +1,7 @@
+const DOM_API_KEY = 'overrideDOM';
+const STRING_API_KEY = 'overrideText';
+
+module.exports = {
+    DOM_API_KEY,
+    STRING_API_KEY
+};
```

Comment:
I do not see a reason for this file, we are not on front-end => optimizsation by ES modules is not relevant here. I would keep it in the ensure-valid-value module.

Reply:
It is a solution for cyclic dependency issue. This is used in many places, sometimes it will cause issues when tried to be imported.
![image](https://user-images.githubusercontent.com/46347627/105845961-2ef62e80-5fe4-11eb-81a0-904fb2e19c8f.png)